docs/README.md

```diff
@@ -12,8 +12,8 @@ The documentation is build on [Mintlify](https://www.npmjs.com/package/mintlify)
 
 ```
 # make sure you're in `/docs` folder, where `mint.json` and `package.json` files are
-npm ci
-npm run dev
+npm i -g mintlify
```

Comment:
not sure about the -g flag, should it be global?

Reply:
The mintlify in-repo isn't working as expected because of "sharp".
A global installation of mintlify allows us to use the command without relying on repo-dependencies that cause it to break.